packages/site/src/types/custom.d.ts

```diff
@@ -1,13 +1,13 @@
-/* eslint-disable*/
-
+// eslint-disable-next-line spaced-comment
 /// <reference types="react-scripts" />
```

Comment:
Can't we remove this comment?

Reply:
If we remove this line, eslint will replace

```ts
/// <reference types="react-scripts" />
```

with

```ts
// / <reference types="react-scripts" />
```

But I just found a rule to change this behaviour, I'll try it.

Reply:
Fixed 🎉